src/test/java/com/example/moviemanager/service/TmdbClientTest.java

```diff
@@ -0,0 +1,121 @@
+package com.example.moviemanager.service;
+
+import com.example.moviemanager.configuration.TmdbConfigProperties;
+import com.example.moviemanager.service.model.FindMovieInfo;
+import com.example.moviemanager.service.model.MovieDetails;
+import com.example.moviemanager.service.model.MovieDetailsResponse;
+import com.example.moviemanager.service.model.MovieType;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.Collections;
+import java.util.List;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TmdbClientTest {
+
+    private static final String TV_SHOWS_URL = "/3/search/tv?api_key=43d5d872d7dc32845ab997b8v25987f7" +
```

Comment:
I don't like that both urls contain query information specific for one specific test. Either have only base url together with api key here (then you at least have it reusable) or have the whole url in test.